app/views/admin/organizations/index.html.slim

```diff
@@ -0,0 +1,21 @@
+- content_for :title do
+  - 'Co-ops and Organizations'
+
+= render 'layouts/admin/tables'
+
+= grid(@organizations_grid, show_filters: :when_filtered) do |g|
+
+  - g.column name: 'ID', attribute: 'id'
+
+  - g.column name: 'Name', attribute: 'name'
+
+  - g.column name: 'City', attribute: 'city'
+
+  - g.column name: 'Country' do |organization|
+    - organization.country.name
+
+  - g.column name: 'Created', attribute: 'created_at' do |organization|
+    - organization.created_at
+
+  - g.column name: 'Updated', attribute: 'updated_at' do |organization|
+    - organization.updated_at
```

Comment:
For date formatting.

Per Fuzzy, wice grid can handle default date formats. So blocks may not be needed.